spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/TcpConfigOutboundGatewayTests-context.xml

```diff
@@ -37,6 +38,9 @@
 							   serializer="crLfSerializer"
 							   deserializer="crLfSerializer"/>
 
+	<ip:tcp-inbound-gateway id="gatewayCrLf2"
+		connection-factory="crLfServer2"
+		request-channel="toSA" />
```

Comment:
Would mind explaining this trick?

I guess it is a part of:
> Fix tcpOutboundGatewayInsideChain test to use the correct port.

But isn't clear how it works.
Thanks.